.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
What if the questions are regarding the project, in that case, we would need to have blank templates right?

Reply:
For questions it's generally recommended to use [Discussion](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions) threads for better SEO & discoverability on Search Engines. Let me know if it still needs to be changed though?

Reply:
cool makes sense

Reply:
So sometime i report broken links to projects, where will that classify 😅 ?

Reply:
> So sometime i report broken links to projects, where will that classify 😅 ?

Yeah makes sense. In that case in I'll configure the Bug Report template accordingly if that works? For context, I'll make changes to #4 to consider broken links in any of the documentation, READMEs, etc. Does that work then?

Reply:
Yup 👍🏽